benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -442,6 +444,7 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "bulkScanCaseReference", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "onlinePanel", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "caseBundles", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "historicalBundles", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
```

Comment:
This line is not needed since the role derives from caseworker-sscs which has CRUD access